app/views/users/_user.html.erb

```diff
@@ -0,0 +1,11 @@
+<%= link_to user_path(user) do %>
+<li class= "user ml-2 mt-4 flex items-center gap-12">
+<div class = "user-image">
+<%= image_tag user.photo, alt: "#{user.name}-avatar", class:'w-40 h-36 object-cover object-center rounded-[50%]' %>
+</div>
+<div class="user-info w-12/12 border border-red flex flex-col">
+<h3 class='username text-3xl ml-2'><%= user.name %>
+<span class='self-end mr-2'><%= user.posts_counter%></span>
+</div>
+</li>
```

Comment:
You did very well on this project. Well done 👏🏾 

- [ ] The previous reviewer mentioned something that is important and I believe you should work on it. The project requirements expect that you follow the project wireframe as closely as possible. if you compare your work with the wireframe you will notice a few differences. 

Your work | 
---- | 
![image](https://user-images.githubusercontent.com/84156714/230105682-95e241f3-2c69-4333-bdc0-13af1df60554.png)